router/dnslog.go

```diff
@@ -45,5 +45,6 @@ func readAndParseDNS(filename string) (map[string][]string, error) {
 			}
 		}
 	}
+	g.Close()
```

Comment:
This commit is called "changes to useful var name", but you also added `g.Close()` as part of it. Really they should have been two commits if possible.